app/util/Enumerable.scala

```diff
@@ -45,6 +45,11 @@ object Enumerable {
           JsError("error.invalid")
       }
 
+    /* util.Enumerable[A] in method writes is never used however there is a
+      name clash between defined and inherited member:
+      - implicit def writes[A]: play.api.libs.json.Writes[A] in trait Implicits and
+      - implicit lazy val writes: play.api.libs.json.Writes[models.dto.InternationalAddressChoiceDto] at line 44
+     */
     implicit def writes[A: Enumerable]: Writes[A] =
```

Comment:
I'm not sure this is even used. I took it out and everything compiles fine. However, it's hard to be sure with implicits so maybe best to leave it in.

Reply:
It must be use because if I remove the Enumerable I have a compiler warning/error.